gpflux/layers/basis_functions/fourier_features/base.py

```diff
@@ -36,6 +36,13 @@ def __init__(self, kernel: gpflow.kernels.Kernel, n_components: int, **kwargs: M
         super(FourierFeaturesBase, self).__init__(**kwargs)
         self.kernel = kernel
         self.n_components = n_components
+        if isinstance(kernel, gpflow.kernels.MultioutputKernel):
+            self.is_multioutput = True
+            self.num_latent_gps = kernel.num_latent_gps
+        else:
+            self.is_multioutput = False
+            # self.num_latent_gps = None
```

Comment:
Can this not be set to 1, for consistency?

Reply:
yep